airflow/serialization/serialized_objects.py

```diff
@@ -644,6 +644,7 @@ def deserialize_operator(cls, encoded_op: Dict[str, Any]) -> Union[BaseOperator,
                 deps=tuple(),
                 is_dummy=False,
                 template_fields=(),
+                template_ext=(),
```

Comment:
This one needs to be “reset later” (as mentioned in the comment a few lines above) like `template_fields`, but I couldn’t find where this is done specifically. What do I need to do to make it work?

Reply:
So long as it's included in `MappOperator.get_serialized_fields()` it should get included in the output, and automatically set on deserialization.

Reply:
And yeah, running the tests it looks like it is already handled

```
1615  ->     assert op.mapped_kwargs['bash_command'] == literal                                                                                                                                                                                        
1616         assert op.partial_kwargs['executor_config'] == {'dict': {'sub': 'value'}}                                                                                                                                                                 
(Pdb++) p op
MappedOperator(task_type=BashOperator, task_id='a', partial_kwargs={'executor_config': {'dict': {'sub': 'value'}}}, mapped_kwargs={'bash_command': [1, 2, {'a': 'b'}]}, dag=None)
(Pdb++) p op.template_ext
['.sh', '.bash']
(Pdb++) 
['.sh', '.bash']
(Pdb++) p op.template_fields
['bash_command', 'env']
(Pdb++) 
['bash_command', 'env']
```

Reply:
I like it when things automatically work.